compair/tests/api/test_assignment.py

```diff
@@ -777,7 +777,7 @@ def test_get_all_status(self):
                     self.assertFalse(status['comparisons']['available'])
                     self.assertEqual(status['comparisons']['count'], compare_count_result)
                     self.assertEqual(status['comparisons']['left'],
-                        assignment.total_comparisons_required - compare_count_result)
+                        max(0, assignment.total_comparisons_required - compare_count_result))
```

Comment:
why was this changed? (if compare_count_result is less than zero, there is a problem somewhere)

Reply:
The existing function `_submit_all_possible_comparisons_for_user` compares all possible pairs for a given user without checking how many comparison is required for the assignment.

Will change that logic and fix affected tests.

Reply:
alright, `_submit_all_possible_comparisons_for_user` is working correctly (sorry for not catching that)